src/devices/cellular.c

```diff
@@ -359,7 +360,7 @@ static bool auth_telem_stream(struct serial_buffer *sb,
 
         /* Parse the incoming JSON */
         jsmn_parser parser;
-        jsmntok_t toks[TELEM_AUTH_JSMN_TOKENS];
+        jsmntok_t toks[TELEM_AUTH_JSMN_TOKENS] = {};
```

Comment:
Yes. it will zero initialize the array, and we need that for searching nodes.  

There's a node search routine that expects to find null tokens at the end, signalling it should stop searching. Without initializing to zero, it was causing it spill beyond the end and crashing. 

initializing it to {} seemed cleaner than an explicit memset call.